tests/ReportConfigTest.php

```diff
@@ -1,6 +1,7 @@
 <?php
 use Octri\ReportCounts\ReportConfig,
-    PHPUnit\Framework\TestCase;
+    PHPUnit\Framework\TestCase,
+    Prophecy\PhpUnit\ProphecyTrait;
```

Comment:
A deprecation message was logged that the version of Prophecy built into PHPUnit will be removed in PHPUnit 10. This protects us against failures when that happens.